_pkgdown.yml

```
@@ -72,6 +72,7 @@ reference:
   contents:
   - between
   - case_when
+  - case_match
```

```suggestion
  - case_match
  - case_when
```

Based on the alphabet 😛